src/main/java/seedu/address/model/recipe/Recipe.java

```diff
@@ -120,7 +128,8 @@ public String toString() {
         // TODO: edit if needed
         StringBuilder sb = new StringBuilder();
         sb.append(getName())
-                .append(String.format("; Portions: %2f", getPortion()));
+                .append(String.format("; Completion time: %d, Portions: %2f",
+                        getCompletionTime(), getPortion()));
```

Comment:
How long is a unit of completion time? I think it should be reflected here.
```suggestion
                .append(String.format("; Completion time: %d (mins), Portions: %2f",
                        getCompletionTime(), getPortion()));
```

Reply:
Yep seems gud LFG